GetStream/stream-chat-react-native
# fix: solves multiple crashes in the Sample app, that were related to setBlurType

…using the removed setBlurType

## 🎯 Goal
`setBlurType` that was removed caused a crash when calling it in the on an item in the Photos and Videos section in the sample app. Documentation is also have been updated

## 🛠 Implementation details

All usages of `setBlurType` were removed

## 🎨 UI Changes

N/A

## 🧪 Testing

- run sample app
- go to a channel
- click on the channel's avatar (top-right)
- click on Photos and Videos
- click on a photo
- the photo should be visible in the gallery

## ☑️ Checklist

- [ ] Commits follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) spec
